svelte.typescript.preprocessor.js

```diff
@@ -0,0 +1,55 @@
+import fs from 'fs'
+import path from 'path'
+import {promisify} from 'util'
+
+import chalk from 'chalk'
+import * as ts from 'typescript'
+
+import tsconfig from './tsconfig.json'
+
+const removeFile = promisify(fs.unlink)
```

Comment:
removeFIle and unlink file has different meaning, 
how about naming thie `unlinkAsync`